python/benchmarks/coco/transforms.py

```diff
@@ -0,0 +1,662 @@
+#!/usr/bin/env python
+#
+# Data augmentation during object detection training.
+#
+# Copied from https://raw.githubusercontent.com/pytorch/vision/main/references/detection/transforms.py
```

Comment:
really interesting these are not available as library code

Reply:
Yea, the official torchvision one is transform applied to image only.  But these data augmentation transforms apply to both image and target. for example, flip needs to flip both image and bbox.